src/main/java/Crystal.java

```diff
@@ -0,0 +1,298 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import exceptions.InvalidCommandException;
+import exceptions.IncompleteCommandException;
+import exceptions.IllegalCommandException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.Todo;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Crystal {
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskCount = 0;
+    private static final int TODO_CHAR_COUNT = 4;
+    private static final int DEADLINE_CHAR_COUNT = 8;
+    private static final int EVENT_CHAR_COUNT = 5;
+
+    public static void horizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void sayHello() {
+        horizontalLine();
+        System.out.println("Hola! I'm Crystal.\n"
+            + "What can I do for you today?");
+        horizontalLine();
+    }
+
+    public static void sayBye() {
+        horizontalLine();
+        System.out.println("Adios, hasta luego!");
+        horizontalLine();
+    }
+
+    public static void list() {
+        horizontalLine();
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println((i + 1) + ". " + tasks.get(i));
+        }
+        horizontalLine();
+    }
+
+    public static void mark(int taskNumber) {
```

Comment:
Could you be more descriptive in this method name (together with the functionality with the unmark method)?